apply/applydriver/local.go

```diff
@@ -149,7 +149,7 @@ func (c *Applier) scaleCluster(mj, md, nj, nd []string) error {
 		return nil
 	}
 
-	logger.Info("Start to scale this cluster")
+	logger.Info("Start to scale this cluster, current cluster: master %s, worker %s", c.ClusterCurrent.GetMasterIPList(), c.ClusterCurrent.GetNodeIPList())
```

Comment:
What if it has 100 nodes, the iplist will very long

Reply:
> What if it has 100 nodes, the iplist will very long

How about changing the log level of the current cluster to DEBUG mode?

Reply:
logger.Info("Start to scale this cluster")
logger.Debug("current cluster:.....")